sorting/MergeSort.r

```diff
@@ -0,0 +1,31 @@
+ merge.merge <- function(a, b) {
```

Comment:
Use proper naming convention for function names (as well as variables). (for example, `merge.merge` doesn't make much sense!)